coverage-tests/configuration/eg_configuration.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  name: 'eyes_selenium_java',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.java3/pullRequest?prid=7879168)
```suggestion
  name: "eyes_selenium_java",
```